.github/workflows/qemu.yml

```
@@ -49,6 +50,115 @@ jobs:
+        uses: actions/checkout@v4
+
       - name: Run install script
         run: |
-          sudo ./scripts/install.sh --email="${{ env.ADMIN_EMAIL }}" --password="${{ env.ADMIN_PASSWORD }}"
\ No newline at end of file
+          cmd="./scripts/install.sh"
+          
+          if [ -n "${{ matrix.email }}" ]; then
+            cmd="$cmd --email=${{ matrix.email }}"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)